lib/index.ts

```diff
@@ -171,6 +171,9 @@ function checkInternal(jsonStr: string): {
   ast: DocumentNode | undefined;
   ctx: Ctx;
 } {
+  if (typeof jsonStr !== 'string') {
+    throw 'Input must be a string. (Use JSON.stringify first.)';
```

Comment:
```suggestion
    throw new TypeError('Input must be a string. (Use JSON.stringify first.)');
```

TypeError seems appropriate for this case, in which the wrong type is passed.